aliases/available/git.aliases.bash

```diff
@@ -86,7 +86,7 @@ alias ggui='git gui'
 # home
 alias ghm='cd "$(git rev-parse --show-toplevel)"' # Git home
 # appendage to ghm
-if ! _command_exists gh; then
+if ! _binary_exists gh; then
```

Comment:
If the user creates their own `gh` alias, its not our place to override it ... `_command_exists` seems correct here to me.

Reply:
But we shouldn't override it with the GitHub CLI binary. Should we?